### PR TITLE
Acknowledge .mjs files

### PR DESCRIPTION
**Pull request by Avaq**
Hi! `sanctuary-lint` assumes that files must end in `js` to be linted. I've added the new `mjs` extension to it.

### CODE REVIEW

**Review by davidchambers — APPROVED** [review]
LGTM. Let's wait until sanctuary-js/sanctuary-style#22 has been merged and published before merging and publishing this pull request.

**Comment by davidchambers** [comment]
Please rebase this branch when you have a minute, Aldwin. :)

**Comment by Avaq** [comment]
Rebased.

**Pull request merged**
